common/api/core-frontend.api.md

```diff
@@ -5989,6 +5989,12 @@ export class IdleTool extends InteractiveTool {
     static toolId: string;
 }
 
+// @beta (undocumented)
+export interface ILocalCache {

```

Comment:
> Do not use I as a prefix for interface names.

Please make sure you have reviewed our [coding guidelines](https://www.itwinjs.org/learning/guidelines/typescript-coding-guidelines/#names)